### PR TITLE
Fix dashboard issues found in demo

### PR DESCRIPTION
**Pull request by AdamAdHocTeam**
## Description of change

There where a few issues found in the demo:

- Remove "Grants" from y-axis label of TTA Hrs chart.
- Fix Date Range warning message to be: "Reports are available from 09/01/2020. Use the format MM/DD/YYYY."
- Date Range selector prevents a Start Date of 09/01/2020.
- Dashboard Overview widget should show dollar sign icon '$' for Grants.

## How to test

Log into site the following should be fixed:

- Remove "Grants" from y-axis label of TTA Hrs chart.
- Fix Date Range warning message to be: "Reports are available from 09/01/2020. Use the format MM/DD/YYYY."
- Date Range selector prevents a Start Date of 09/01/2020.
- Dashboard Overview widget should show dollar sign icon '$' for Grants.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-258

### Every PR

- [x] Meets issue criteria
- [x] JIRA ticket status updated
- [x] Code is meaningfully tested
- [x] Meets accessibility standards (WCAG 2.1 Levels A, AA)
~- [ ] API Documentation updated~
~- [ ] Boundary diagram updated~
~- [ ] Logical Data Model updated~
~- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions~

### Production Deploy

~- [ ] Staging smoke test completed~

### After merge/deploy

- [ ] Update JIRA ticket status

### CODE REVIEW

**Review by SarahJaine — COMMENTED** [review]
@AdamAdHocTeam Can you also include the minor changes that Matt H came up with in review of the merge dev > staging? https://github.com/HHS/Head-Start-TTADP/pull/519

**Review by SarahJaine — COMMENTED** [review]
All this is great! I am seeing one weird thing though. If I try to select 09/01/2020 as the start date I get an the "choose a valid date error" 09/02/2020 is the first date that works.
Screen Shot 2021-08-05 at 5 34 12 PM

File: frontend/src/components/DateRangePicker.js [hunk 1]
```diff
@@ -66,12 +66,12 @@ function DateRangePicker({
   return (
     <>
       { showDateError ? (
-        <div className="usa-alert margin-1" role="alert">
+        <div className="usa-alert usa-alert--warning margin-1" role="alert">
           <div className="usa-alert__body">
             <p className="usa-alert__text">
-              Please enter a valid start date
+              Reports are available from 09/01/2020.
```
[the remark this answers is not in the document]
Comment (AdamAdHocTeam) [reply frontend/src/components/DateRangePicker.js:72]: Ok I have put in a fix for this.

**Review by jasalisbury — APPROVED** [review]
:+1:

**Pull request merged**